geomstats/geometry/pre_shape.py

```diff
@@ -753,16 +756,14 @@ def log(self, point, base_point, **kwargs):
             Tangent vector at the base point equal to the Riemannian logarithm
             of point at the base point.
         """
-        sphere = self._space._sphere
-        sphere_embedding_dim = sphere.embedding_space.dim
-        flat_bp = gs.reshape(base_point, (-1, sphere_embedding_dim))
-        flat_pt = gs.reshape(point, (-1, sphere_embedding_dim))
-        flat_log = sphere.metric.log(flat_pt, flat_bp)
+        batch_shape = get_batch_shape(self._space, point, base_point)
```

Comment:
Why "get_batch_shape"? and not "batch_shape". Is this because it is a defined as a property somewhere?

Even in that case, I don't think that we use "get" for the other properties defined in the library: I would remove to keep the coding style consistent, and conciseness.

https://stackoverflow.com/questions/374763/should-i-use-get-set-prefixes-in-python-method-names

Reply:
And we generally don't use "get" in the whole codebase, thus if we can avoid it altogether, it'll be better.

Reply:
this comes from another PR. see [our discussion][https://github.com/geomstats/geomstats/pull/1828#discussion_r1112743175]


I still think having a verb is better for a function (for me a name represents an object, not a callable).  also, as you see in this example, I do ` batch_shape = get_batch_shape(self._space, point, base_point)`. if the function didn't have the verb, I would have to do  `batch_shape_ = batch_shape(self._space, point, base_point)` which is more cumbersome.

maybe we can change this in another PR if you really don't want it, since it is unrelated.